SPECS/kernel/CVE-2020-27777.nopatch

```diff
@@ -0,0 +1 @@
+CVE-2020-27777 - already patched in linux-msft-5.4.83 stable kernel
```

Comment:
Please add the upstream commit id and the stable commit id so we have a track record of the patch.

Reply:
Fixed, thanks!